docs/source/using/troubleshooting.adoc

```diff
@@ -230,3 +230,16 @@ If you try to use Minishift with Hyper-V using an external virtual switch while
 Cause: Hyper-V networking might not route the network traffic in both directions properly when connected to a VPN.
 
 Workaround: Disconnect from the VPN and try again after stopping the VM from the Hyper-V manager.
+
+[[root-filesystem-exceeds-overlay-size]]
+== The root filesystem of the Minishift VM exceeds overlay size
+
+Installing additional packages or copying large files to the root filesystem of the Minishift VM might exceed the allocated overlay size and lock the Minishift VM.
+
+Cause: The Minishift VM root filesystem contains core packages that are configured to optimize running the Minishift VM and the application containers.
```

Comment:
It should be just containers as user might run diagnostics tools in containers. s/the application containers/containers/